blog/2020-10-12-ambrosia-alpha-release.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Trade Bitcoin and DAI with COMIT"
+author: franck
+author_url: https://github.com/d4nte
+author_image_url: https://avatars1.githubusercontent.com/u/300805
+tags: [bitcoin,dai,atomic swap,ambrosia]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# A Change of Direction
+
+Last year, we spent time and effort creating tools, such as [`create-comit-app` and `comit-scripts`](https://github.com/comit-network/create-comit-app/) to allow developers to build their own COMIT Apps.
+We organised local meetups in Sydney and even a hackathon in Singapore.
+We had a number of developers excited about the technology and possibilities that COMIT bring in but very few committed to build an app.
+
+We know realize that before other build successful apps with our tools, we should demonstrate what can be done with COMIT ourselves.
+
+# Ambrosia
+
+<img alt="Ambrosia" src={useBaseUrl('blog/assets/images/2020-10-12/ambrosia.png')} />
+
+This is why we spent the last few months working on [Ambrosia](https://github.com/comit-network/ambrosia/), an UI for trading in the COMIT network.
+Ambrosia allows you to trade BTC/DAI in a decentralised manner. No third party to trust, no central orderbook, no KYC.
+It uses [cnd](https://github.com/comit-network/ambrosia/comit-rs) has a bakend.
+
+The software has some rough edges, but we wanted to get it out there to have your feedback. Here are some choices we want your input on:
+
+## Custody
+
+Not only you do not need to trust a third party with your fund when trading, even our software is non-custodial.
+Ambrosia never holds your crypto, it interfaces with your hardware wallet (nano ledger s only) to make payments.
+
+We believe that custody and security are important criteria to our users, hence we also assume that users are running their own [Bitcoin Core](https://github.com/bitcoin/bitcoin) node.
+
+## Trading pair
+
+The COMIT Network Daemon (cnd) supports various trading pairs: BTC/ETH, BTC/ERC20, Lightning BTC/ETH, etc.
+However, we have limited Ambrosia to the BTC/DAI pair as we know that Bitcoin/USD.. are usually the most popular pair.
+
+## Decentralised Orderbook
+
+At this stage, Ambrosia can only take offers.
+We have a [nectar](https://github.com/comit-network/comit-rs) instance that runs and acts as a maker.
+
+## Trying it out
+
+The software is currently in an alpha state, bugs are known and expected.
+
+If you want to try out, we strongly recommend you have a chat with us so we can assist:
+
+-   Email: [team@coblox.tech](mailto:team@coblox.tech)
+-   Matrix chat: [#comit:matrix.org](https://matrix.to/#/!HYBOPcopXgKbEnEELc:matrix.org?via=matrix.org&via=privacytools.io) (e2e encryption enabled)
+-   Twitter: [@comit_network](https://twitter.com/comit_network)
+
+If you want to try it yourself, you need 2 components:
```

Comment:
If you add this section you might also want to add that Ambrosia depends on Ledger Nano S being present. 
And the full-node requirement.

Reply:
Yeah, I hesitated, will add.